pkg/doc/verifiable/credential.go

```diff
@@ -421,9 +603,80 @@ func loadCredentialSchema(url string, client *http.Client) ([]byte, error) {
 	return gotBody, nil
 }
 
-// JSONBytes converts Verifiable Credential to JSON bytes
-func (vc *Credential) JSONBytes() ([]byte, error) {
-	rawCred := &rawCredential{
+// JWT serializes Credential to signed JWT.
+func (vc *Credential) JWT(
+	signatureAlg JWTAlgorithm,
+	privateKey interface{},
```

Comment:
TODO - need to refactor this. Private keys will never leave the wallet, therefore it cannot be passed in as a parameter to anything outside of the wallet itself.

Reply:
OK, I need more input about this, let's discuss this tomorrow!

Reply:
Don't worry about it for now - create an issue and add a TODO here that references it.

Reply:
#339